soc/arm/nxp_kinetis/ke1xf/Kconfig.defconfig.series

```diff
@@ -81,6 +81,13 @@ config ADC_MCUX_ADC12
 	default y
 	depends on ADC
 
+if DAC
+
+config DAC_MCUX_DAC32
+	default y
```

Comment:
instead of wrapping with `if DAC..endif`, just add `depends on DAC`

Reply:
Fixed.